src/cpp/src/prompt_lookup/continuous_batching_for_prompt_lookup.cpp

```diff
@@ -83,4 +83,16 @@ void ContinuousBatchingPipeline::ContinuousBatchingForPromptLookupImpl::generate
     }
 }
 
+void ContinuousBatchingPipeline::ContinuousBatchingForPromptLookupImpl::clear_requests() {
+    drop_requests();
```

Comment:
why do we need this `clear_requests` method? 
looks like instead of `clear_requests` you can call `drop_requests`, but move it to public section. 

not tested, but maybe something like:
```
class ContinuousBatchingPipeline::ContinuousBatchingForPromptLookupImpl {
public:
   using ContinuousBatchingPipeline::ContinuousBatchingImpl::drop_requests;
};
```